services/events/manage.py

```diff
@@ -61,6 +61,12 @@ def recreate_db():
     db.create_all()
     db.session.commit()
 
+@manager.command
+def create_db():
```

Comment:
Allows the creation of a database; avoids dropping it if it already exists. After accidentally running `recreate_db` against the staging RDS, I've opted to add this as part of this PR.